src/main/java/com/amazon/opendistroforelasticsearch/sql/query/maker/AggMaker.java

```diff
@@ -555,6 +555,9 @@ private DateHistogramAggregationBuilder dateHistogram(MethodField field) throws
                     case "interval":
                         dateHistogram.dateHistogramInterval(new DateHistogramInterval(kv.value.toString()));
                         break;
+                    case "fixed_interval":
+                        dateHistogram.fixedInterval(new DateHistogramInterval(kv.value.toString()));
+                        break;
```

Comment:
What is this change about?

Reply:
Changes regarding ES and painless API are due to the old ones we used are deprecated. The previous testing with internal cluster didn't complain about this but it became problem now with external cluster.